newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -27,8 +31,10 @@ import {
   assetStoreHomePageState,
 } from './AssetStoreNavigator';
 import { type ChosenCategory } from '../UI/Search/FiltersChooser';
+import _ from 'lodash';
```

Comment:
I think it's not recommended to import the whole library like this, because it can increase the bundle size.
let's do `import shuffle from 'lodash/shuffle';`

(found this article https://www.blazemeter.com/blog/import-lodash-libraries)

Reply:
Thanks!